src/cdomains/intDomain.ml

```diff
@@ -1530,10 +1530,33 @@ module Enums : S = struct
         ) Invariant.none ns
 end
 
+module type IntDomTupleConf = sig
+  val def_exc: unit -> bool
+  val interval: unit -> bool
+  val cinterval: unit -> bool
+  val enums: unit -> bool
+end
+
+module IntDomTupleStdConf: IntDomTupleConf = struct
+  let lookup s = get_bool @@ "ana.int."^s
+  let def_exc () = lookup "def_exc"
+  let interval () = lookup "interval"
+  let cinterval () = lookup "cinterval"
+  let enums () = lookup "enums"
+end
+
+module IntDomTupleInvariantConf: IntDomTupleConf = struct
```

Comment:
Replace content with
~~~ocaml
include IntDomTupleStdConf
let def_exc () = true
~~~

Reply:
Done, in e5333b75677a22989c534cccf996999aa17aa12c